servers.yml

```diff
@@ -0,0 +1,39 @@
+test:
```

Comment:
Does yml allow comments? If it does, we should add some basic documentation for users how to deal with this file and explain the non-obvious fields such as capacity.

Reply:
It's possible for comments, but I don't know add what and how docmuentation.